### PR TITLE
add telemetry to load_screen_fails_and_caregivers_into_screen_follow_up

### PR DESCRIPTION
**Pull request by ChemiKyle**
Addresses issue #61 

Edits from maintainers are allowed, change content as needed.

### CODE REVIEW

File: load_screen_fails_and_caregivers_into_screen_follow_up.R [hunk 1]
```diff
@@ -63,6 +63,14 @@ rc_upload <- screen_fails_and_caregivers %>%
...
-redcap_write(rc_upload,
+result <- redcap_write(rc_upload,
         redcap_uri = Sys.getenv("URI"),
         token = Sys.getenv("SCREEN_TOKEN"))
+
+message <- paste0("Uploaded rows: ", nrow(rc_upload),
+        "\nAffected records: ", result$records_affected_count
+        )
+
+email_subject <- paste("Screen fails loaded", Sys.getenv("INSTANCE"), script_run_time, "| Success", write_data$success)
```
Comment (pbchase) [inline load_screen_fails_and_caregivers_into_screen_follow_up.R:74]:
```suggestion
email_subject <- paste("Screen fails loaded", Sys.getenv("INSTANCE"), script_run_time, "| Success", result$success)
```

You had the wrong variable for the results of the `redcap_write`

Comment (ChemiKyle) [reply load_screen_fails_and_caregivers_into_screen_follow_up.R:74]: Stole code from myself and didn't respect the variable names

File: load_screen_fails_and_caregivers_into_screen_follow_up.R [hunk 2]
```diff
@@ -63,6 +63,14 @@ rc_upload <- screen_fails_and_caregivers %>%
...
 # specify local context for data upload
 #load_dot_env('local.env')
-redcap_write(rc_upload,
+result <- redcap_write(rc_upload,
         redcap_uri = Sys.getenv("URI"),
         token = Sys.getenv("SCREEN_TOKEN"))
+
+message <- paste0("Uploaded rows: ", nrow(rc_upload),
+        "\nAffected records: ", result$records_affected_count
+        )
```
Comment (pbchase) [inline load_screen_fails_and_caregivers_into_screen_follow_up.R:72]:
```suggestion
message <- paste0(nrow(rc_upload), " screen fail records in input file\n",
        result$records_affected_count, " records written to ", Sys.getenv("SCREEN_PROJECT_TITLE")
        )
```

The message content I suggested was robotic. I think this is better suited for the puny humans

Comment (ChemiKyle) [reply load_screen_fails_and_caregivers_into_screen_follow_up.R:72]: 🤖 `THAT MESSAGE CONTENT WAS CREATED ORGANICALLY BY MY FAT-BASED ELECTROCHEMICAL COMPUTING CENTER. OFFENSE TAKEN. RELATION PENALTY 3.000000000000000001`

Comment (pbchase) [reply load_screen_fails_and_caregivers_into_screen_follow_up.R]: Really, I thought it took my level of expertise to write something like that.

**Pull request merged**